src/script/lua_api/l_playermeta.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+Minetest
+Copyright (C) 2013 celeron55, Perttu Ahola <celeron55@gmail.com>
```

Comment:
Outdated copypasta information

Reply:
fixed in local, just waiting for build to finish

Reply:
done